sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/SqlIoTest.java

```diff
@@ -540,6 +540,43 @@ public void compare(String query, String expected, boolean optimize) {
         this.addRustTestCase(query, compiler, circuit, streams);
     }
 
+    // Unsure what to do with `data`
+    // We already create an InputOutputPair from the preparedInputs and the expected result
+    private void runtimeFailInner(boolean optimize, String query, String expected, String message, InputOutputPair... _data) {
+        DBSPCompiler compiler = this.testCompiler(optimize);
+        this.prepareData(compiler);
+        compiler.compileStatement("CREATE VIEW VV AS " + query);
+        if (!compiler.options.languageOptions.throwOnError)
+            compiler.throwIfErrorsOccurred();
+        compiler.optimize();
+        DBSPCircuit circuit = getCircuit(compiler);
+        DBSPType outputType = circuit.getOutputType(0);
+        DBSPZSetLiteral.Contents result = this.parseTable(expected, outputType);
```

Comment:
there is no expected output, so there should be no parseTable since the table is always empty.
You can just synthesize an empty ZSetLiteral for the output with the outputType.